scripts/asm-installer/asm_vm

```diff
@@ -631,6 +646,10 @@ EOF
   elif [[ -n "${KEY_FILE}" ]]; then
     fatal "Couldn't find key file ${KEY_FILE}."
   fi
+
+  if [[ ! -z "${SOURCE_INSTANCE_TEMPLATE}" ]] && [[ ! -z "${IMAGE_FAMILY}" ]]; then
```

Comment:
replace with `-n`

Reply:
As there will only be one mesh agent for both debian and centos which can automatically detect the os system and use correct envoy path

Reply:
Then why do we need --image-family flag at all? User can simply use source instance template to choose their intended OS.

Reply:
but there is one cases that the user doesn't have any source instance template. They want directly create a new template in one step and I think that is the user's requirement.

Reply:
I just add a limitation: At most one of these parameters may be specified